README.md

```diff
@@ -74,33 +77,59 @@ There are two ways to install DeepRank2:
 
 ### Containerized Installation
 
-In order to try out the package without worrying about your OS and without the need of installing all the required dependencies, we created a `Dockerfile` that can be used for taking care of everything in a suitable container.
+We provide a pre-built Docker image hosted on GitHub Packages, allowing you to use DeepRank2 without worrying about installing dependencies or configuring your system. This is the recommended method for trying out the package quickly.
 
-For this, you first need to install [Docker](https://docs.docker.com/engine/install/) on your system. Then run the following commands. You may need to have sudo permission for some steps, in which case the commands below can be preceded by `sudo`:
+#### Pull and Run the Pre-build Docker Image (Recommended)
+
+- Install [Docker](https://docs.docker.com/engine/install/) on your system, if not already installed.
+- Pull the latest Docker image from GitHub Packages by running the following command:
+
+```bash
+docker pull ghcr.io/deeprank/deeprank2:latest
+```
+
+- Run the container from the pulled image:
+
+```bash
+docker run -p 8888:8888 ghcr.io/deeprank/deeprank2:latest
+```
+
+- Once the container is running, open your browser and navigate to `http://localhost:8888` to access the DeepRank2 application.
+
+From here, you can use DeepRank2, including running the tutorial notebooks. More details about the tutorials can be found [here](https://github.com/DeepRank/deeprank2/blob/main/tutorials/TUTORIAL.md). Note that the Docker container downloads only the raw PDB files required for the tutorials. To generate processed HDF5 files, you will need to run the `data_generation_xxx.ipynb` notebooks. Since Docker containers may have limited memory resources, we reduce the number of data points processed in the tutorials. To fully utilize the package, consider [installing it locally](#localremote-installation).
```

Comment:
```suggestion
From here, you can use DeepRank2, including running the tutorial notebooks. More details about the tutorials can be found [here](https://github.com/DeepRank/deeprank2/blob/main/tutorials/TUTORIAL.md). Note that the Docker container downloads only the raw PDB files required for the tutorials. To generate processed HDF5 files, you will need to run the `data_generation_ppi.ipynb` and/or `data_generation_ppi.ipynb` notebooks. Since Docker containers may have limited memory resources, we reduce the number of data points processed in the tutorials. To fully utilize the package, consider [installing it locally](#localremote-installation).
```